doc/level1.rst

```diff
@@ -275,6 +275,23 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send Autocrypt headers above 10 KiB in size
+(including the Autocrypt: prefix and folding whitespace).
+Conversely, MUAs SHOULD not process Autocrypt headers
+larger than 10 KiB in size.
+
+.. note::
+
+  - An Autocrypt header following this specification
+    will be up to 3 KiB large.
+    Some users import 4096-bit RSA key configurations,
+    which may result in Autocrypt headers of up to 5 KiB.
+  - The 10 KiB limit was chosen to accomodate for these use cases
+    but also avoid headers being truncated or emails being rejected
+    by MTAs with a 10 KiB header size limit.
+    In addition it makes header sizes that are indicative of bugs
+    obvious to implementors.
```

Comment:
This last sentence is a great point, but i'd phrase it the other way around.  It makes it easy for implementors of receiving MUAs to reject buggy headers that happen to be oversize.

Reply:
The intent of my phrasing was that a MUA that sends faulty autocrypt headers will simply fail to work if the oversized headers are rejected. This would surely have caught issues like the ones Enigmail initially had.

I'm unsure what "makes it easier" would even mean in this context for a receiving MUA?

(this was also marked as resolved but not changed nor commented?)

Reply:
Here's my latest take on the last sentence:

> Sending an oversized autocrypt header will result in consistent
    behaviour accross various clients.
    This way the issue can be detected more easily.

@Valodim does it capture your thought?
@dkg works for you?